ansible/library/extract_log.py

```diff
@@ -90,9 +90,14 @@ def extract_line(directory, filename, target_string):
         file = gzip.GzipFile(path)
     else:
         file = open(path)
-    result = None
+    result = []
```

Comment:
This change is not required. The empty list will be overwritten.

Reply:
Agreed, Either way is fine. The append function in caller takes both None and [] and the result is the same (in case no entry were found).